src/morris_sensitivity.jl

```diff
@@ -24,4 +32,24 @@ function generate_design_matrix(prob::DEProblem,p_range,p_steps,k = 10)
         B[j] = cur_p
     end
     B
+end
+
+function calculate_spread(matrix::Array{Array{Float64}})
```

Comment:
Just leave this duck typed. It'll fail anyways since it should be `Array{<:Array{Float64}}` since `Array` is abstract without a dimension.